### PR TITLE
[stable/kafka-manager] configurable liveness and readiness probes

### LINKED ISSUE #16377: [stable/kafka-manager] Liveness and Readiness probes should be configurable
**Describe the bug**
Everytime kafka-manager restarts, it needs time to load its previous state which means the default liveness probe timeout is too short.

**Which chart**: stable/kafka-manager

**What happened**: liveness probes are failing after the pod restarts

**What you expected to happen**: I should be able to increase liveness probe value

**How to reproduce it** (as minimally and precisely as possible):
- Deploy chart on a large kafka cluster with a lot of partitions and enable consumer polling
- Restart kafka manager

### PR DESCRIPTION
**Pull request by agolomoodysaada**
Signed-off-by: moody <moody.saada@agolo.com>

#### What this PR does / why we need it:
Make probes configurable

#### Which issue this PR fixes
  - fixes #16377 

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

**Comment by agolomoodysaada** [comment]
@giacomoguiulfo 
@ssalaues

**Comment by ssalaues** [comment]
@agolomoodysaada It looks like the PR needs rebase off the master branch and bump the version again. After that it should be good to merge.

File: stable/kafka-manager/values.yaml [hunk 1]
```diff
@@ -12,6 +12,19 @@ serviceAccount:
   ##
   name: ""
 
+livenessProbe:
+  httpGet:
+    path: /api/health
+    port: kafka-manager
+  # initialDelaySeconds: 60
```
Comment (ssalaues) [inline stable/kafka-manager/values.yaml]: If the defaults aren't adequate for typical installs, they should be adjusted.

**Comment by agolomoodysaada** [comment]
@ssalaues , thank you! 
I made the changes as requested.

**Pull request merged**